### ISSUE TITLE
Post-It could be scrollable?

### ISSUE THREAD

**Issue by Snaid1** [Snaid1/BHUD-Blishpad#9]
There was a request to have the Post-It be scrollable to account for long texts. BlishHUD doesn't natively support this and it will need to be looked into further to find a solution (or wait for BlishHUD core to add scrollability to Multiline Textboxes)

**Comment by patrickmaciel** [comment]
![image](https://user-images.githubusercontent.com/671670/235234228-4cd5c72c-b651-4139-897b-79a1ec2ceaf7.png)

I really want a scroll here, because now I cant create another notes, I cant see the notes on the bottom of the page.

**Comment by Snaid1** [comment]
Yep, that's a problem. I'll make that a top priority item when I have time
to work on Blishpad.

I've got several enhancements I can do now that BlishHUD supports more
things...
